x-pack/legacy/plugins/cross_cluster_replication/public/app/app.js

```diff
@@ -34,18 +34,16 @@ import {
   FollowerIndexEdit,
 } from './sections';
 
-export class App extends Component {
-  static contextTypes = {
-    router: PropTypes.shape({
-      history: PropTypes.shape({
-        push: PropTypes.func.isRequired,
-        createHref: PropTypes.func.isRequired
-      }).isRequired
-    }).isRequired
-  }
+class AppComponent extends Component {
+  static propTypes = {
+    history: PropTypes.shape({
+      push: PropTypes.func.isRequired,
+      createHref: PropTypes.func.isRequired,
+    }).isRequired,
+  };
 
-  constructor(...args) {
-    super(...args);
+  constructor(props) {
```

Comment:
Does this need to be changed? If possible, I'd prefer to leave this alone to reduce the amount of changes in this PR and lessen the scope of code review.

Reply:
sure, reverted 👍